system-addon/test/unit/lib/SectionsManager.test.js

```diff
@@ -237,6 +252,16 @@ describe("SectionsManager", () => {
         previewImageURL: "image"
       });
     });
+    it("should call PlacesUtils.history.insert", () => {
+      SectionsManager.updateBookmarkMetadata({url: "bar"});
+
+      assert.calledOnce(fakePlacesUtils.history.insert);
+      assert.calledWithExactly(fakePlacesUtils.history.insert, {
+        url: "bar",
+        title: "title",
+        visits: [new Date()]
```

Comment:
Not sure if we need to be testing that the specific added visit date ?

Reply:
Specific date isn't important but if it's missing insert will trow an error https://searchfox.org/mozilla-central/source/toolkit/components/places/PlacesUtils.jsm#1034
Could check the arguments keys instead.